src/main/java/org/benetech/servicenet/matching/counter/UrlSimilarityCounter.java

```diff
@@ -14,48 +13,38 @@
     public static final String HTTPS = "HTTPS://";
     public static final String HTTP = "HTTP://";
     public static final String TRAILING = "/";
-    
-    @Value("${similarity-ratio.weight.url.equal-upper-cased}")
-    private float uppercasedWeight;
 
     @Override
     public float countSimilarityRatio(String url1, String url2, MatchingContext context) {
         if (StringUtils.isBlank(url1) || StringUtils.isBlank(url2)) {
             return NO_MATCH_RATIO;
         }
-        if (areNormalizedAndUpperCasedDifferent(url1, url2)) {
-            return NO_MATCH_RATIO;
-        }
-
-        if (areNormalizedDifferent(url1, url2)) {
-            return uppercasedWeight;
+        if (areNormalizedAndUpperCasedTheSame(url1, url2)) {
+            return COMPLETE_MATCH_RATIO;
         }
 
-        return COMPLETE_MATCH_RATIO;
+        return NO_MATCH_RATIO;
     }
 
     private String normalize(String url) {
```

Comment:
I think you can move the normalization code to some separate utils class and use it in OrganizationConflictDetector too, you can name it NormalizationUtils or something like that